ci/37.yaml

```diff
@@ -13,10 +13,11 @@
    - esda
    - geopandas>=0.4.0
    - giddy
-  #  - ipywidgets
+   - ipywidgets
```

Comment:
This was done on purpose, we need one env without `ipywidgets`.

Reply:
ahhh. Good catch. I'll change that back. Do we want have a `3.7` test with `ipywidgets` and one without? Or just have one without?

Reply:
Also, can't we remove `descartes` from requirements/dependencies since it's part of `geopandas` now?

Reply:
> can't we remove descartes

Yes, but pin to 0.9.0+.

> Do we want have a 3.7 test with ipywidgets and one without? Or just have one without?

I'm fine with having one.

Reply:
> Yes, but pin to 0.9.0+.

I'll just leave as is for now then.

> I'm fine with having one.

👍